apps/frontend-manage/src/components/activities/creation/poll/PollInformationStep.tsx

```
@@ -0,0 +1,136 @@
+              </div>
+              <div className="border-uzh-grey-80 bg-uzh-grey-20 ml-1 hidden h-max w-1/2 flex-col gap-2 rounded-md border border-solid p-3 md:flex">
+                <PropertyList
+                  elements={[
+                    {
+                      icon: faLightbulb,
+                      iconColor: 'text-orange-400',
+                      richText: t.rich('manage.activityWizard.pollUseCase', {
+                        link: (text) => (
+                          <a
+                            href="https://www.klicker.uzh.ch/use_cases/poll/"
+                            target="_blank"
```

Biome (2.4.6)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

(lint/security/noBlankTarget)